### PR TITLE
inbac: Add version 2.1.0

### PR DESCRIPTION
**Pull request by talhagozutok**
Added an initial manifest to install [inbac](https://github.com/weclaw1/inbac), fast interactive image cropping tool.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by talhagozutok** [comment]
Closes #14416

File: bucket/inbac.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "bin": "inbac.exe",
+    "shortcuts": [
+        [
+            "inbac.exe",
+            "inbac"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/weclaw1/inbac"
+    },
```
Comment (aliesbelik) [inline bucket/inbac.json:17]:
```suggestion
    "checkver": "github",
```

**Review by aliesbelik — COMMENTED** [review]
Otherwise looks good.

**Pull request merged**